### DISCUSSION TITLE
How to understand the `supporting functionality`?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#379] [Extension Development QnA] [answered]
When I read the [doc: Panel](https://code.visualstudio.com/api/ux-guidelines/panel) 
> Use for Views that provide supporting functionality

How to understand the `supporting functionality`?

**Comment by isidorn** [comment, marked as answer]
What is not the part of the main user flow. Just checkout what is already in the panel.
